packages/terra-functional-testing/lib/docker/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.7'
+
+services:
+  hub:
+   image: selenium/hub:3.141.59-zirconium
+   ports:
+     - "4444:4444"
+
+  chrome:
```

Comment:
do chrome and firefox depend on hub?

https://docs.docker.com/compose/compose-file/#depends_on

Reply:
Kind of, but depends_on is ignored when deploying docker in swarm mode:

https://docs.docker.com/compose/compose-file/#depends_on

> The depends_on option is ignored when deploying a stack in swarm mode with a version 3 Compose file.